gatsby/src/components/ConferenceHero.js

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable prettier/prettier */
+import React from 'react';
+import { Link } from 'gatsby';
+import styled from 'styled-components';
+import Button from 'react-bootstrap/Button';
+import heroJS from '../images/heroJS.svg';
+import heroBackgr from '../images/heroBackgr.svg';
+
+const Hero = styled.div`
+  background: url(${heroBackgr}) no-repeat 50% 50% / cover;
+  position: relative;
+  justify-content: flex-end;
+  align-items: center;
+  display: flex;
+  min-height: 300px;
+  height: 35vh;
+  font-family: 'Gothic A1', sans-serif;
+  color: #ffffff;   
+  
+  #hero-box {
+    display: flex;
+    justify-content: space-between;
+    margin-right: 150px;
+    width: 480px;
+    height: 240px;
+    padding: 30px;
+    background-color: rgba(22, 29, 14, 0.75);
+    border-radius: 12px;
+  }
+  #hero-text {
+    text-align: left;
+    width: 296px;
+  }
+  #hero-title {
+    font-weight: bold;
+    font-size: 22px;
+  }
+  #hero-subtext {
+    margin-top: 10px;
+    font-size: 18px;
+  }
+  #hero-buttons {
+    margin-top: 15px;
+  }
+  #hero-buttons a {
+    margin-right: 6px;
+  }
+@media (max-width: 768px){
+  #hero-box {
+    margin-right: 20px;
+  }
+  @media (max-width: 520px){
+  justify-content: center;
+  #hero-box {
+    flex-direction: column;
+    align-items: center;
+    margin-right: 0;
+    width: 408px;
+  }
+  #hero-title {
+    font-size: 15px;
+  }
+  #hero-subtext {
+    margin-top: 10px;
+    font-size: 15px;
+  }
+  img{
+    height: 60px;
+  }
+}
+@media (max-width: 300px){
+  #hero-box{
+    padding: 2px;
+  }
+  #hero-text, #hero-buttons{
+    text-align: center;
+  }
+}
+`;
+
+function ConferenceHero() {
+  return (
+    <Hero>
+      <div id='hero-box'>
+        <div id='img-div'>
+          <img src={heroJS} alt='Utah JS Logo' id='hero-box-img' />
```

Comment:
Can we use gatsby-image

You will make a page query to graphql using imageSharp

Reply:
You can do a shearch in the project for where we use imageSharp and gatsby-image to have some examples